### DISCUSSION TITLE
task and worker monitor service

### DISCUSSION THREAD

**Discussion by keinagae** [taskiq-python/taskiq#364] [Q&A]
Hi do we have a service similar to flower to monitor tasks and worker status

**Comment by keinagae** [comment]
also is it possible to get status of workers i am using a middle ware to get infor about tasks but i dont see any option for worker availability

**Reply by s3rius** [reply]
Yes it's possible. You can export prometheus metrics wit, and generally workers are self-healing unless killed manually. So mostly health checks are not required. But I still think of how to implement it correctly so it can be used by orchestration software such as kubernetes.
